axelor-purchase/src/main/resources/views/PurchaseOrder.xml

```diff
@@ -172,7 +172,7 @@
 			<button name="draftBtn" title="Draft" showIf="statusSelect == 5"   onClick="action-purchase-order-record-draft,save"/>
 			<button name="requestedBtn" title="Requested" showIf="statusSelect == 1"   onClick="save,action-purchase-order-method-requested"/>
 			<button name="newVersionBtn" title="New version" showIf="statusSelect == 2" onClick="save,action-purchase-record-newVersion,save" if="__config__.app.getApp('purchase').getManagePurchaseOrderVersion()"/>
-			<button name="validateBtn" title="Validate" showIf="statusSelect == 2"   onClick="action-group-purchase-order-on-validate-actions"/>
+			<button name="validateBtn" title="Validate" showIf="statusSelect == 2"   onClick="action-purchase-order-method-analyticDistributionList-Empty,action-group-purchase-order-on-validate-actions"/>
```

Comment:
add action 'action-purchase-order-method-analyticDistributionList-Empty' in existing action-group.
and change name to 'action-purchase-order-method-check-analytic-distribution-template'